sormas-api/src/main/resources/captions.properties

```diff
@@ -1457,6 +1457,9 @@ personNoContactLinkedToPerson=No contact linked to person
 personLinkToEvents=See events for this person
 personLinkToCases=See cases for this person
 personLinkToContacts=See contacts for this person
+personsReplaceGeoCoordinates=Also replace existing coordinates. Warning: This may data cases where the coordinates where intentionally set differently!
```

Comment:
There's something off with the text here; did you mean to say "This may include cases" or "This may include data" or something like that? In any case I'd replace "may" with "might".